raspi-config

```diff
@@ -4130,6 +4147,7 @@ do_advanced_menu() {
       "A10 Network Install UI" "Select display of bootloader network install UI" \
       "A11 Libliftoff" "Enable/disable libliftoff hardware overlays" \
       "A12 Shutdown Behaviour" "Configure shutdown behavior" \
+      "A13 Logging" "Set storage location for logs" \
```

Comment:
You probably want to add it to all four of the variants of the Advanced menu - there are separate versions of the menu for Pi 5, PI 4, Pi 3 and below and non-Pi platforms - please make sure you add the entry to all which are appropriate.

Reply:
Probably not needed in the x86 version, but I guess it should be in all three of the Pi versions...